doc/source/f2py/allocarr_session.dat

```diff
@@ -14,7 +15,9 @@ foo - Function signature:
  ]
 >>> allocarr.mod.b                             # b is Fortran-contiguous
 array([[ 1.,  2.,  3.],
-       [ 4.,  5.,  6.]],'f')
+       [ 4.,  5.,  6.]], dtype=float32)
+>>> allocarr.mod.b.flags.f_contiguous
+True
 >>> allocarr.mod.b = [[1,2,3],[4,5,6],[7,8,9]] # reallocate/initialize b
```

Comment:
Since this is an actual code session (as in, it's supposed to mirror a session in the repl) I'm not sure I can move the comments elsewhere because it can be confusing. So I'm adding the two spaces. Feel free to suggest some other solution if this seems inadequate.

Reply:
inserting two spaces should be ok